pkgs/applications/misc/tokodon/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, libsForQt5
+, extra-cmake-modules
+, pkg-config
+, libwebsockets
+, qtquickcontrols2
+}:
+
+stdenv.mkDerivation rec {
+  pname = "tokodon";
+  version = "22.04";
+
+  src = fetchFromGitHub {
+    owner = "KDE";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0878zwqfaxy1ikin0dp19rqzi05w0xhhp03i6gqf3l89ml5y986k";
+  };
+
+  nativeBuildInputs = with libsForQt5; [
+    cmake
+    extra-cmake-modules
+    pkg-config
+  ];
+
+  buildInputs = with libsForQt5; [
```

Comment:
Would it be possible to add this package to `pkgs/top-level/qt5-packages.nix` and remove the need for `libsForQt5` prefix?

Reply:
How would that make the `libsForQt5` obsolete?

Reply:
As you can see here https://github.com/NixOS/nixpkgs/blob/350fd0044447ae8712392c6b212a18bdf2433e71/pkgs/development/libraries/signond/default.nix this package is added to `pkgs/top-level/qt5-packages.nix`. It's referencing qmake without the libsForQt5 prefix

Reply:
Is it a native QT application it would be better to go there